src/ProbeMeasurements.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+#define WATER_PROBE_ACQUANATIVA_AP3_PROBE_MEASUREMENTS_HPP
+
+#include <base/Float.hpp>
+#include <base/Temperature.hpp>
+#include <base/Time.hpp>
+
+namespace water_probe_acquanativa_ap3 {
+    struct ProbeMeasurements {
+        base::Time time;
+        
+        /** Water dissolved oxygen concentration in Kg/L */
+        float concentration = base::unknown<float>();
+
+        /** Water dissolved oxygen % saturation */
+        float saturation = base::unknown<float>();
+
+        /** Water temperature in K */
+        base::Temperature temperature;
+
+        /** Water pH */
+        float ph = base::unknown<float>();
+
+        /** Water conductivity in S/cm*/
```

Comment:
Normalize all units. Here, S/m

Reply:
conversion is right in Driver.cpp... just the comment is wrong here